orttraining/orttraining/test/graph/gradient_graph_builder_test.cc

```diff
@@ -1015,50 +1184,83 @@ TEST(GradientGraphBuilderTest, TrainingSession_WithPipeline) {
   // 100 -> 199: sub 0
   // 200 -> 299: sub 1
   // 300 -> 399: sub 2
-
-  const std::vector<EventsPerBatch> events = {
-      // batch 0
-      {{0, 1, 2, 3},
-       {{-1, 100}, {-1, 200}, {-1, 300}, {202, 203}, {104, 105}}},
+  // so for sub 0, the schedule is:
+  //   F0(-1, 100), F1(100,101), F2(101,102)...
+  // for sub 1, the schedule is:
+  //   F0(-1, 200), F1(200,201), F2(201,202)...
+  // for sub 2, the schedule is:
+  //   F0 B0(-1, 300), F1 B1(300, 301), F2 B2(301, 302)...
+  // Note the chart above is timeline view, execution plan needs to change it to batch view
```

Comment:
Thanks for adding the extra comments :). For complicated designs, viewer sometimes cannot understand the matter as thorough as the writer, that's when some extra comment helps!